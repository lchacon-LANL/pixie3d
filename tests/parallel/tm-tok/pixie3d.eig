0.007812	-5.42024e-08	1.23879e-07	8.97985e-07	1.92776e-16	1.27518e-17	2.65166e-16	-1.29904e-06	-2.685e-08	-7.70778e-18	9.77699e-18	1.92812e-16	-8.97752e-07	-3.23947e-08	-1.29285e-06	-1.27867e-16	1.25695e-18
0.023437	-1.52212e-07	3.68634e-07	8.97536e-07	1.92416e-16	3.69614e-17	8.2105e-17	-1.25512e-06	-9.10038e-08	-2.10572e-17	2.75805e-17	1.92715e-16	-8.95848e-07	-9.27239e-08	-1.30102e-06	-3.08909e-16	8.18463e-19
0.039062	-2.42307e-07	6.13239e-07	8.95695e-07	1.90419e-16	6.33951e-17	3.00165e-16	-1.11576e-06	-1.61512e-07	-3.31705e-17	4.57572e-17	1.92263e-16	-8.87454e-07	-1.44282e-07	-1.26215e-06	-4.68565e-17	2.72198e-18
0.054687	-3.22147e-07	8.46618e-07	8.90729e-07	1.84231e-16	8.08947e-17	6.19812e-17	-9.03228e-07	-2.36873e-07	-4.3792e-17	6.27299e-17	1.90933e-16	-8.64225e-07	-1.82631e-07	-1.19408e-06	-3.0209e-16	-1.61709e-18
0.070312	-3.88355e-07	1.06433e-06	8.80289e-07	1.70865e-16	1.06765e-16	3.01009e-16	-6.29637e-07	-3.18171e-07	-5.23316e-17	7.72247e-17	1.88047e-16	-8.15514e-07	-2.05629e-07	-1.1009e-06	7.85646e-17	8.84934e-20
0.085937	-4.37369e-07	1.26042e-06	8.61729e-07	1.46749e-16	1.15284e-16	1.08743e-17	-3.12085e-07	-4.05077e-07	-5.81503e-17	9.04278e-17	1.82744e-16	-7.30673e-07	-2.13786e-07	-9.86385e-07	-3.25261e-16	-7.14753e-18
0.101562	-4.66075e-07	1.42749e-06	8.32528e-07	1.09626e-16	1.42479e-16	3.08414e-16	2.59725e-08	-4.95258e-07	-6.07715e-17	9.73274e-17	1.74362e-16	-6.01855e-07	-2.10273e-07	-8.54977e-07	2.7962e-16	-7.47057e-18
0.117188	-4.72447e-07	1.55723e-06	7.90757e-07	5.79404e-17	1.41877e-16	-6.66516e-17	3.5492e-07	-5.84069e-07	-5.96881e-17	1.01745e-16	1.62016e-16	-4.2674e-07	-2.00352e-07	-7.12399e-07	-4.39138e-16	-1.13976e-17
0.132812	-4.56134e-07	1.64145e-06	7.35508e-07	-6.19007e-18	1.76609e-16	3.39079e-16	6.42062e-07	-6.64789e-07	-5.51552e-17	9.42271e-17	1.45757e-16	-2.10683e-07	-1.90241e-07	-5.65464e-07	5.55939e-16	-1.53673e-17
0.148438	-4.18873e-07	1.6734e-06	6.67243e-07	-8.03115e-17	1.6682e-16	-1.69788e-16	8.5637e-07	-7.29347e-07	-4.736e-17	7.86823e-17	1.25074e-16	3.23026e-08	-1.8561e-07	-4.21673e-07	-7.6211e-16	3.04082e-18
0.164062	-3.64586e-07	1.64899e-06	5.87967e-07	-1.56607e-16	2.16616e-16	4.19682e-16	9.74515e-07	-7.69465e-07	-3.73255e-17	5.5345e-17	1.01291e-16	2.80276e-07	-1.90058e-07	-2.88452e-07	9.30743e-16	-1.45024e-17
0.179688	-2.99064e-07	1.56784e-06	5.0118e-07	-2.31013e-16	1.92306e-16	-3.02252e-16	9.86348e-07	-7.7815e-07	-2.63023e-17	9.53337e-19	7.42102e-17	5.06271e-07	-2.04e-07	-1.72271e-07	-1.46255e-15	6.59407e-17
0.195312	-2.29216e-07	1.43415e-06	4.11526e-07	-2.92596e-16	2.63052e-16	5.8711e-16	8.98226e-07	-7.51429e-07	-1.45246e-17	-1.33209e-17	4.65322e-17	6.83318e-07	-2.24365e-07	-7.77725e-08	1.52028e-15	-7.92966e-18
0.210938	-1.61976e-07	1.25733e-06	3.24168e-07	-3.37148e-16	2.09126e-16	-4.8277e-16	7.32525e-07	-6.89882e-07	-6.15278e-18	-1.54975e-16	1.75526e-17	7.90576e-07	-2.45348e-07	-7.15637e-09	-2.75848e-15	2.19529e-16
0.226562	-1.03155e-07	1.05173e-06	2.4399e-07	-3.61433e-16	3.08046e-16	8.88526e-16	5.23018e-07	-5.99329e-07	5.10665e-18	-3.38186e-17	-8.03313e-18	8.1868e-07	-2.60098e-07	4.00143e-08	2.62268e-15	-5.71501e-17
0.242188	-5.65368e-08	8.35085e-07	1.74809e-07	-3.77562e-16	1.98462e-16	-7.59389e-16	3.07102e-07	-4.90128e-07	2.08197e-18	-5.15567e-16	-3.42382e-17	7.72255e-07	-2.62822e-07	6.65095e-08	-4.97478e-15	5.83311e-16
0.257812	-2.34794e-08	6.25815e-07	1.18835e-07	-2.97401e-16	3.47378e-16	1.38897e-15	1.17308e-07	-3.7511e-07	2.26374e-17	3.21224e-16	-5.4098e-17	6.68503e-07	-2.50568e-07	7.67051e-08	4.80655e-15	-4.16087e-16
0.273438	-3.10656e-09	4.39793e-07	7.65018e-08	-4.65718e-16	1.33932e-16	-1.2256e-15	-2.52181e-08	-2.66769e-07	-1.60848e-17	-1.57147e-15	-6.7973e-17	5.32286e-07	-2.24017e-07	7.56307e-08	-8.69697e-15	1.53889e-15
0.289062	7.03232e-09	2.87691e-07	4.66861e-08	4.95181e-17	4.10851e-16	2.19267e-15	-1.13301e-07	-1.74672e-07	5.90654e-17	1.87827e-15	-9.8848e-17	3.89497e-07	-1.87061e-07	6.80521e-08	9.00271e-15	-1.72044e-15
0.304688	1.00878e-08	1.73712e-07	2.72179e-08	-9.0029e-16	-3.87501e-17	-2.03855e-15	-1.52116e-07	-1.03927e-07	-8.48145e-17	-4.36951e-15	-6.05775e-17	2.61078e-07	-1.45411e-07	5.78067e-08	-1.49549e-14	3.891e-15
0.320312	9.10229e-09	9.59952e-08	1.54908e-08	9.45334e-16	6.00219e-16	3.47246e-15	-1.54589e-07	-5.5055e-08	1.54443e-16	5.93474e-15	-1.75646e-16	1.59506e-07	-1.04846e-07	4.74991e-08	1.65401e-14	-4.9967e-15
0.335938	6.46815e-09	4.83774e-08	8.99982e-09	-2.05378e-15	-4.35573e-16	-3.4322e-15	-1.35953e-07	-2.50711e-08	-2.36446e-16	-1.02525e-14	2.70678e-17	8.83847e-08	-6.97891e-08	3.85411e-08	-2.52112e-14	8.72734e-15
0.351562	3.71968e-09	2.27341e-08	5.69168e-09	2.65573e-15	1.09487e-15	5.48297e-15	-1.09363e-07	-9.16079e-09	3.49774e-16	1.37286e-14	-3.39038e-16	4.45136e-08	-4.25821e-08	3.14168e-08	2.89509e-14	-1.1314e-14
0.367188	1.60139e-09	1.11055e-08	4.1014e-09	-4.14534e-15	-1.23082e-15	-5.70556e-15	-8.36434e-08	-2.28728e-09	-4.96472e-16	-2.01616e-14	2.43033e-16	2.10851e-08	-2.35469e-08	2.60357e-08	-4.09611e-14	1.69394e-14
0.382812	2.92465e-10	7.09563e-09	3.31749e-09	5.24882e-15	2.09518e-15	8.53147e-15	-6.30624e-08	-2.94844e-10	6.63234e-16	2.57107e-14	-6.43325e-16	1.06648e-08	-1.15687e-08	2.20447e-08	4.7456e-14	-2.12499e-14
0.398438	-3.37477e-10	6.37519e-09	2.85278e-09	-7.08242e-15	-2.58944e-15	-9.17151e-15	-4.84919e-08	-3.46111e-10	-8.65513e-16	-3.40063e-14	6.24988e-16	7.13671e-09	-4.85498e-09	1.90503e-08	-6.30649e-14	2.87239e-14
0.414062	-5.22769e-10	6.53114e-09	2.49235e-09	8.49806e-15	3.72917e-15	1.29063e-14	-3.89148e-08	-8.69e-10	1.07831e-15	4.11737e-14	-1.1188e-15	6.43863e-09	-1.57913e-09	1.67243e-08	7.23542e-14	-3.45477e-14
0.429688	-4.79439e-10	6.51828e-09	2.16685e-09	-1.04915e-14	-4.57664e-15	-1.40787e-14	-3.27615e-08	-1.19605e-09	-1.31363e-15	-5.05774e-14	1.18243e-15	6.41105e-09	-2.6155e-10	1.48425e-08	-9.12239e-14	4.34229e-14
0.445312	-3.55594e-10	6.10859e-09	1.86726e-09	1.19692e-14	5.9848e-15	1.87951e-14	-2.86187e-08	-1.20026e-09	1.55041e-15	5.84831e-14	-1.76043e-15	6.21183e-09	1.02904e-10	1.32624e-08	1.02715e-13	-5.01851e-14
0.460938	-2.31134e-10	5.40137e-09	1.60079e-09	-1.38916e-14	-7.11244e-15	-2.05425e-14	-2.55671e-08	-9.60313e-10	-1.79088e-15	-6.80117e-14	1.89246e-15	5.70986e-09	9.69695e-11	1.19068e-08	-1.23937e-13	5.97727e-14
0.476562	-1.36185e-10	4.59325e-09	1.37185e-09	1.5208e-14	8.70888e-15	2.62317e-14	-2.30752e-08	-6.17593e-10	2.02477e-15	7.56913e-14	-2.53286e-15	5.03117e-09	5.66795e-14	1.07249e-08	1.36583e-13	-6.67581e-14
0.492188	-7.45194e-11	3.80969e-09	1.18008e-09	-1.68779e-14	-9.99737e-15	-2.85161e-14	-2.09429e-08	-2.57084e-10	-2.24533e-15	-8.44492e-14	2.70934e-15	4.33116e-09	-8.14435e-11	9.69134e-09	-1.58938e-13	7.63525e-14
0.507812	-3.83637e-11	3.13633e-09	1.02043e-09	1.78927e-14	1.16636e-14	3.5092e-14	-1.90592e-08	5.68989e-11	2.45385e-15	9.11384e-14	-3.38496e-15	3.69998e-09	-1.28048e-10	8.77847e-09	1.7155e-13	-8.29227e-14
0.523438	-1.91241e-11	2.57339e-09	8.87799e-10	-1.92198e-14	-1.29824e-14	-3.78096e-14	-1.74264e-08	3.2433e-10	-2.63745e-15	-9.85189e-14	3.57978e-15	3.1732e-09	-1.45782e-10	7.97437e-09	-1.93831e-13	9.19727e-14
0.539062	-9.18227e-12	2.12879e-09	7.76196e-10	1.98854e-14	1.46008e-14	4.51315e-14	-1.59883e-08	5.30854e-10	2.80701e-15	1.03779e-13	-4.26496e-15	2.74297e-09	-1.49646e-10	7.2564e-09	2.05369e-13	-9.76903e-14
0.554688	-4.50629e-12	1.76283e-09	6.82067e-10	-2.08624e-14	-1.58388e-14	-4.81396e-14	-1.47605e-08	7.04165e-10	-2.94584e-15	-1.09512e-13	4.45569e-15	2.39522e-09	-1.45858e-10	6.61952e-09	-2.26619e-13	1.05867e-13
0.570312	-1.97789e-12	1.47674e-09	6.0124e-10	2.12003e-14	1.73189e-14	5.60414e-14	-1.36749e-08	8.33913e-10	3.0718e-15	1.13216e-13	-5.1304e-15	2.10743e-09	-1.41975e-10	6.04357e-09	2.36355e-13	-1.10517e-13
0.585938	-8.84307e-13	1.23304e-09	5.32129e-10	-2.18721e-14	-1.83997e-14	-5.91863e-14	-1.27517e-08	9.48956e-10	-3.16564e-15	-1.17302e-13	5.30087e-15	1.86911e-09	-1.36537e-10	5.52954e-09	-2.55967e-13	1.177e-13
0.601562	1.0979e-16	1.04253e-09	4.71768e-10	2.19396e-14	1.96869e-14	6.75019e-14	-1.1921e-08	1.03274e-09	3.24986e-15	1.19545e-13	-5.95183e-15	1.66569e-09	-1.33466e-10	5.05853e-09	2.6352e-13	-1.21247e-13
0.617188	3.30141e-13	8.73546e-10	4.19719e-10	-2.23718e-14	-2.05706e-14	-7.06402e-14	-1.12151e-08	1.11279e-09	-3.30333e-15	-1.22158e-13	6.09227e-15	1.49418e-09	-1.29005e-10	4.63654e-09	-2.81215e-13	1.27464e-13
0.632812	8.71365e-13	7.43312e-10	3.73573e-10	2.22359e-14	2.16436e-14	7.9221e-14	-1.0566e-08	1.16791e-09	3.35136e-15	1.23155e-13	-6.71222e-15	1.34495e-09	-1.26952e-10	4.24471e-09	2.86495e-13	-1.29989e-13
0.648438	1.02972e-12	6.23e-10	3.33577e-10	-2.24945e-14	-2.23198e-14	-8.22321e-14	-1.00169e-08	1.22564e-09	-3.37134e-15	-1.24546e-13	6.81805e-15	1.21824e-09	-1.23083e-10	3.8932e-09	-3.02245e-13	1.35352e-13
0.664062	1.4853e-12	5.32949e-10	2.97583e-10	2.22161e-14	2.31811e-14	9.09554e-14	-9.50018e-09	1.26195e-09	3.39009e-15	1.2455e-13	-7.4042e-15	1.10641e-09	-1.21623e-10	3.56236e-09	3.05361e-13	-1.36993e-13
0.679688	1.58629e-12	4.4574e-10	2.66301e-10	-2.23573e-14	-2.36592e-14	-9.37457e-14	-9.06681e-09	1.30494e-09	-3.38393e-15	-1.24983e-13	7.47446e-15	1.01113e-09	-1.18091e-10	3.26585e-09	-3.19299e-13	1.41651e-13
0.695312	1.99623e-12	3.83235e-10	2.37712e-10	2.19853e-14	2.43262e-14	1.02516e-13	-8.64871e-09	1.32861e-09	3.38024e-15	1.24241e-13	-8.02702e-15	9.25808e-10	-1.17037e-10	2.9828e-09	3.20472e-13	-1.42562e-13
0.710938	2.05353e-12	3.19031e-10	2.12859e-10	-2.20526e-14	-2.46268e-14	-1.05019e-13	-8.30238e-09	1.36172e-09	-3.35459e-15	-1.23955e-13	8.063e-15	8.5296e-10	-1.13756e-10	2.72995e-09	-3.32814e-13	1.46666e-13
0.726562	2.41506e-12	2.75692e-10	1.89776e-10	2.16233e-14	2.51236e-14	1.13767e-13	-7.95892e-09	1.3768e-09	3.33466e-15	1.2268e-13	-8.58392e-15	7.8676e-10	-1.13017e-10	2.48492e-09	3.32313e-13	-1.46993e-13
0.742188	2.42658e-12	2.27723e-10	1.69756e-10	-2.16481e-14	-2.52723e-14	-1.15938e-13	-7.67897e-09	1.40322e-09	-3.29474e-15	-1.21875e-13	8.58812e-15	7.30219e-10	-1.09948e-10	2.26726e-09	-3.43301e-13	1.5068e-13
0.757812	2.73625e-12	1.9787e-10	1.5084e-10	2.11878e-14	2.56249e-14	1.24616e-13	-7.39281e-09	1.41243e-09	3.26433e-15	1.20237e-13	-9.08021e-15	6.78054e-10	-1.09463e-10	2.05291e-09	3.41399e-13	-1.5055e-13
0.773438	2.70421e-12	1.61511e-10	1.34513e-10	-2.11908e-14	-2.5648e-14	-1.26431e-13	-7.16416e-09	1.4343e-09	-3.21327e-15	-1.19075e-13	9.0557e-15	6.33571e-10	-1.06582e-10	1.86398e-09	-3.51269e-13	1.53935e-13
0.789062	2.96543e-12	1.41235e-10	1.188e-10	2.07193e-14	2.58811e-14	1.35007e-13	-6.92259e-09	1.43946e-09	3.17846e-15	1.17205e-13	-9.52206e-15	5.91885e-10	-1.06308e-10	1.6747e-09	3.48218e-13	-1.53455e-13
0.804688	2.89754e-12	1.13276e-10	1.0534e-10	-2.07119e-14	-2.58031e-14	-1.36454e-13	-6.73408e-09	1.45825e-09	-3.11628e-15	-1.1581e-13	9.4721e-15	5.56464e-10	-1.036e-10	1.50952e-09	-3.57186e-13	1.5663e-13
0.820312	3.11848e-12	9.98479e-11	9.21245e-11	2.02447e-14	2.59388e-14	1.44911e-13	-6.52764e-09	1.46062e-09	3.08519e-15	1.13803e-13	-9.91568e-15	5.22723e-10	-1.03504e-10	1.34095e-09	3.53209e-13	-1.55884e-13
0.835938	3.02483e-12	7.80306e-11	8.09187e-11	-2.02325e-14	-2.57814e-14	-1.45991e-13	-6.3709e-09	1.47733e-09	-3.00668e-15	-1.12269e-13	9.84357e-15	4.94211e-10	-1.00958e-10	1.19563e-09	-3.61455e-13	1.58921e-13
0.851562	3.21498e-12	6.95238e-11	6.96774e-11	1.97792e-14	2.58388e-14	1.5432e-13	-6.19247e-09	1.47778e-09	2.99306e-15	1.10179e-13	-1.0267e-14	4.66579e-10	-1.01013e-10	1.04435e-09	3.56745e-13	-1.5797e-13
0.867188	3.10526e-12	5.2239e-11	6.02673e-11	-1.97725e-14	-2.56119e-14	-1.5504e-13	-6.06116e-09	1.49311e-09	-2.88178e-15	-1.08603e-13	1.01759e-14	4.43409e-10	-9.86212e-11	9.15785e-10	-3.6442e-13	1.60931e-13
0.882812	3.27327e-12	4.72796e-11	5.06037e-11	1.93175e-14	2.56367e-14	1.63239e-13	-5.90531e-09	1.49226e-09	2.91684e-15	1.06381e-13	-1.05818e-14	4.20539e-10	-9.88087e-11	7.79078e-10	3.59139e-13	-1.59784e-13
0.898438	3.15555e-12	3.33668e-11	4.26399e-11	-1.93561e-14	-2.52359e-14	-1.63611e-13	-5.79453e-09	1.50669e-09	-2.7204e-15	-1.04888e-13	1.04736e-14	4.01562e-10	-9.65665e-11	6.64783e-10	-3.66332e-13	1.62722e-13
0.914062	3.30891e-12	3.09655e-11	3.42504e-11	1.88883e-14	2.54811e-14	1.71682e-13	-5.65707e-09	1.50497e-09	2.89587e-15	1.02372e-13	-1.08665e-14	3.82433e-10	-9.68736e-11	5.40456e-10	3.60749e-13	-1.61364e-13
0.929688	3.19053e-12	1.95906e-11	2.74642e-11	-1.87453e-14	-2.44492e-14	-1.71725e-13	-5.5629e-09	1.51884e-09	-2.4591e-15	-1.00168e-13	1.07406e-14	3.66716e-10	-9.47784e-11	4.38412e-10	-3.67255e-13	1.63685e-13
0.945312	3.33622e-12	1.90603e-11	2.01156e-11	1.91844e-14	2.46673e-14	1.79657e-13	-5.4406e-09	1.51654e-09	2.92812e-15	9.90976e-14	-1.11224e-14	3.50183e-10	-9.51625e-11	3.24688e-10	3.61883e-13	-1.63305e-13
0.960938	3.22136e-12	9.75152e-12	1.43099e-11	-1.64831e-14	-2.5906e-14	-1.79415e-13	-5.36008e-09	1.53002e-09	-2.26107e-15	-8.67518e-14	1.10083e-14	3.36371e-10	-9.31183e-11	2.33238e-10	-3.68706e-13	1.5941e-13
0.976562	3.34774e-12	1.06635e-11	7.84218e-12	2.00557e-14	1.10092e-14	1.87177e-13	-5.25297e-09	1.52755e-09	1.89695e-15	9.46061e-14	-1.12899e-14	3.20792e-10	-9.3264e-11	1.28661e-10	3.58183e-13	-1.63272e-13
0.992188	3.19944e-12	3.40145e-12	2.89872e-12	-2.10683e-14	-4.87954e-14	-1.86552e-13	-5.18862e-09	1.54102e-09	-4.09493e-15	-5.58604e-14	1.14155e-14	3.11158e-10	-9.10224e-11	4.63489e-11	-3.84971e-13	1.4699e-13
1.007810	3.19944e-12	3.40145e-12	-2.89872e-12	-2.10683e-14	-4.87954e-14	-1.74929e-10	-5.10818e-09	1.54103e-09	-4.09493e-15	-5.58604e-14	-1.14155e-14	3.11158e-10	-9.10224e-11	1.24658e-10	3.39585e-13	-7.02706e-14
