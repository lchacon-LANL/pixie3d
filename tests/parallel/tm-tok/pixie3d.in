 &datin
    ilevel   = 0
    test     = t

    nxd      = 64
    nyd      = 32
    nzd      = 1

    numerical_grid = f

    xmax     = 1.
    ymax     = 0.
    zmax     = 1.

    bcs      = 'spt','def','per','per','per','per'

    eta      = 1d-4
    nu       = 1d-4
    dd       = 1d-4
    chi      = 0d-3
    
    equil    = '2dkai'

    coords   = 'hel'

    dlambda  = 0.6
    gparams  = 1d0,-0.3,3.333333,2d0,0d0

    gamma    = 2d0

    dt       = 1d-3
    tmax     = 1d-2
    dstep    = 1d-2

    pvx      = 1d-5
    nh2      = 1
!    odd      = f,f,f
!    random   = f
    prt_file = "pixie3d-eigen.eig"

    cnfactor = 0.5

    restart  = f
    timecorr = f

    source   = t

    precon   = 'id'

    method   = 1
    tolgm    = 0.8
    iguess   = 0

    solenoidal = t
    nc_eom_f   = t
    nc_eom_v   = f
    adiabatic  = t
    use_p_eom  = f

    debug    = f
    debug_it = 1

    check_grid = f
 /

 &graphdef
    sel_diag  = 2,3,4,5,6,7,12,15,17,18,19,20,22,23
    sel_graph = 1,2,3,4,5,6,7,8,11
    dplot = 1d-2
    prof_conf = 1,'r',1,10,1
    cont_conf = 3,'x','y',1,1,1
 /

# Aspect ratio = 0.3

    gparams  = 1d0,-0.3,3.3333333,2d0,0d0

# Aspect ratio = 1.0

    gparams  = 1d0,-1.0,1.0,2d0,0d0
