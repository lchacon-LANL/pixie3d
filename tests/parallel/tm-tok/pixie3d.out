Processor:  0; Grid limits:  1 32  1 32  1  1
Processor:  1; Grid limits: 33 64  1 32  1  1
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 Reading KAI TM perturbations
 
 ***********************************************
 ****** PIXIE3D Extended MHD code, V. 0.9 ******
 ******       Developer: L. Chacon        ******
 ******  Los Alamos National Laboratory   ******
 ******       Los Alamos, NM 87545        ******
 ******           LA-CC 07-005            ******
 ***********************************************
 
 ****************** NOTICE ******************
 Using non-conservative JxB     part of EOM
 Using non-conservative grad(p) part of EOM
 ****************** NOTICE ******************

  Grid mesh:................   64x   32x    1
  Number of grid levels:....    4,    4,    1
  Time correction...........   F
  External source...........   T
  Numerical grid............   F
  Advection scheme..........   ZIP    
  TS scheme................. Theta    
 
   itime    time     dt    vx_max vy_max vz_max GMRES  Newton  CN factor
      1    0.001  1.0E-03   0.000  0.000  0.000     5     4     0.500
      2    0.002  1.0E-03   0.000  0.000  0.000     5     4     0.500
      3    0.003  1.0E-03   0.000  0.000  0.000     5     4     0.500
      4    0.004  1.0E-03   0.000  0.000  0.000     6     5     0.500
      5    0.005  1.0E-03   0.000  0.000  0.000     5     4     0.500
      6    0.006  1.0E-03   0.000  0.000  0.000     5     4     0.500
      7    0.007  1.0E-03   0.000  0.000  0.000     7     5     0.500
      8    0.008  1.0E-03   0.000  0.000  0.000     6     4     0.500
      9    0.009  1.0E-03   0.000  0.000  0.000     7     4     0.500
     10    0.010  1.0E-03   0.000  0.000  0.000     8     5     0.500

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt  Whst/GMRES
   10       4.3        5.9        1.4        0.0
rank 0 in job 264  moscato.lanl.gov_38714   caused collective abort of all ranks
  exit status of rank 0: return code 0 
